src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4328,7 +4328,7 @@ namespace HVACVariableRefrigerantFlow {
                         ShowContinueError("... DX cooling coil air inlet node = " + NodeID(CCoilInletNodeNum));
                         ErrorsFound = true;
                     }
-                    if (VRFTU(VRFTUNum).HeatingCoilPresent) {
+                    if (VRFTU(VRFTUNum).HeatingCoilPresent && !VRFTU(VRFTUNum).SuppHeatingCoilPresent) {
```

Comment:
If there no supplemental heating coil, then the VRF terminal unit outlet node should match the heating coil outlet node name.